start.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import os
+
+HELP_STR = """Syntax:
+    python ./start.py action mcVersion metaUrl
+Example:
+    python ./start.py yarn 1.8.9
+    python ./start.py yarn 1.7.10 https://meta.legacyfabric.net
+    python ./start.py publishlocal 1.8.9"""
+
+def main():
+    args = sys.argv
+    if len(set(["--help", "-h"]) & set(args))>0:
```

Comment:
Why didn't you use the [argparse](https://docs.python.org/3/library/argparse.html#module-argparse) module?

Reply:
I didn't feel that it's necessary, and still don't